project/src/main/kotlin/org/ionproject/core/access_control/PDP.kt

```diff
@@ -0,0 +1,112 @@
+package org.ionproject.core.access_control
```

Comment:
As much as I like snake case, we should rename the package to `accessControl` for consistency.

Reply:
consider it as done